funnel/forms/sync_ticket.py

```diff
@@ -25,8 +27,27 @@
     'TicketTypeForm',
 ]
 
+
 BOXOFFICE_DETAILS_PLACEHOLDER = {'org': 'hasgeek', 'item_collection_id': ''}
 
+FORM_SCHEMA_PLACEHOLDER = {
+    "fields": [
+        {
+            "name": "title",
+            "title": "Display title",
+            "description": "An explanation",
+            "type": "string",
+        },
+        {"name": "checkbox", "title": "Checkbox", "type": "boolean"},
+        {
+            "name": "choice",
+            "title": "Choose one",
+            "type": "string",
+            "enum": ["First choice", "Second choice", "Third choice"],
```

Comment:
The "type" here is bothering me. We are not doing any validation, so it doesn't make sense to specify this as "string". It could be "select", and "enum" key could be named "choices".